message/message.yml

```diff
@@ -0,0 +1,415 @@
+swagger: '2.0'
+info:
+  description: Send and receive messages between systems using a publish-subscribe model.
+  version: '1'
+  title: SystemLink Message Service
+  contact:
+    name: National Instruments
+    url: 'https://www.ni.com/systemlink'
+    email: support@ni.com
+basePath: /nimessage
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  apiKeyAuth:
+    type: apiKey
+    name: x-ni-api-key
+    in: header
+  basicAuth:
+    type: basic
+security:
+  - apiKeyAuth: []
+  - basicAuth: []
+
+responses:
+  Unauthorized:
+    description: Not authorized
+    headers:
+      WWW_Authenticate:
+        description: Information for how to authenticate (only sent by SystemLink Server).
+        type: string
+  Error:
+    description: Error
+    schema:
+      description: Error response
+      title: ErrorResponse
+      type: object
+      properties:
+        error:
+          $ref: '#/definitions/Error'
+
+parameters:
+  Token:
+    in: path
+    name: token
+    description: The unique session ID
+    type: string
+    required: true
+
+paths:
+  /v1/sessions:
+    post:
+      tags:
+        - Sessions
+      summary: Create a session
+      description: Creates a unique messaging session that can send and receive messages.
+      operationId: CreateSession
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/SessionToken'
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        403:
+          description: Reached the maximum number of sessions
+        default:
+          $ref: '#/responses/Error'
+  /v1/sessions/{token}:
+    parameters:
+      - $ref: '#/parameters/Token'
+    delete:
+      tags:
+        - Sessions
+      summary: Delete a session
+      description: Deletes a messaging session. Messaging sessions that are not explicitly deleted are automatically cleaned up after five minutes of inactivity.
+      operationId: DeleteSession
+      responses:
+        204:
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/sessions/{token}/subscriptions/subscribe:
+    parameters:
+      - $ref: '#/parameters/Token'
+    post:
+      tags:
+        - Topics
+      summary: Subscribe to a topic
+      description: Subscribes to a topic to receive all messages broadcast with that topic name.
```

Comment:
I think the endpoint they're referring to is the node that gets dropped, so that seems reasonable. I think both are okay in their respective contexts.